python/Makefile

```diff
@@ -95,17 +95,23 @@ push_conda:
 	@echo "Alternatively use anaconda upload to publish on own channel"
 
 setup_linter:
-	pip install black==20.8b1
+	pip install black==20.8b1 isort==5.7.0
 
 fmt:
 	black \
 		./ ../testing ../operator/helm ../operator/hack \
-		--exclude "(testing/scripts/proto|seldon_core/proto/|.eggs|.tox)"
+		--exclude "(proto|seldon_core/proto/|.eggs|.tox)"
```

Comment:
Note sure about this one, I think there was a reason why why excluded `testing/scripts`.
@axsaucedo @adriangonz?

Reply:
@RafalSkolasinski we exclude the `proto` folder within `testing/scripts`. It would be good to double check whether that expression also captures that or not. Otherwise `black` is gonna go crazy formatting the generating code for the protobufs.

Reply:
Yeah, make total sense for `protos`. Was just wondering if there is something else in `testing/scripts` we would like to leave unmoddified.

Reply:
Okay, cool, I don't see any black re-formats in the protos. Only `isort` touched some imports, added separate comment for that: https://github.com/SeldonIO/seldon-core/pull/2927#discussion_r570865145

Reply:
I was thinking formatting pb2 files would be fine too. I don't see any disadvantage with that as long as it's always formatted whenever they're regenerated. But I'll follow the convention and ignore them here.

Reply:
https://github.com/SeldonIO/seldon-core/tree/master/testing/scripts/proto is empty now except for a `.keep` file ...